backend/app/utils/comment_helpers.py

```diff
@@ -0,0 +1,16 @@
+"""Helper functions for comments."""
```

Comment:
This just can't be called a helper. Create a new directory called "service"  and under service create a new file called comment_service.py and extract and add this code.  

Service: if it involves a network call, a database access etc.
Helper: if it's a simple operation like string formatting, simple calculation or input validation